durabletask-azuremanaged/durabletask/azuremanaged/client.py

```diff
@@ -12,22 +12,13 @@ class DurableTaskSchedulerClient(TaskHubGrpcClient):
     def __init__(self, *,
                  host_address: str,
                  taskhub: str,
-                 secure_channel: Optional[bool] = True,
-                 metadata: Optional[list[tuple[str, str]]] = None,
-                 token_credential: Optional[TokenCredential] = None):
+                 token_credential: TokenCredential = None,

```

Comment:
Let's remove the ` = None`. Let's make customers always be explicit about the `token_credential` parameter.